packages/flutter/lib/src/cupertino/text_field.dart

```diff
@@ -778,6 +781,20 @@ class CupertinoTextField extends StatefulWidget {
   /// {@macro flutter.services.TextInputConfiguration.enableIMEPersonalizedLearning}
   final bool enableIMEPersonalizedLearning;
 
+  /// {@macro flutter.widgets.text_selection.TextMagnifierConfiguration.intro}
+  ///
+  /// {@macro flutter.widgets.magnifier.intro}
+  ///
+  /// {@macro flutter.widgets.text_selection.TextMagnifierConfiguration.details}
+  ///
+  /// By default, builds a [CupertinoTextMagnifier] on iOS nothing on all other
```

Comment:
I wonder if its okay to build the `CupertinoTextMagnifier` on all mobile platforms. I think we build the `CupertinoTextSelectionControls`/ `CupertinoDesktopTextSelectionControls` https://github.com/flutter/flutter/blob/928dd48c50c0272234822637d34169d52e6ac583/packages/flutter/lib/src/cupertino/text_field.dart#L1197  for all platforms